transmart-core-api/src/main/groovy/org/transmartproject/core/users/PatientDataAccessLevel.groovy

```diff
@@ -0,0 +1,35 @@
+package org.transmartproject.core.users
+
+/**
+ * Describes a level of read access to the patient data.
+ *
+ * Enum values declaration order specifies {@link Enum#compareTo(PatientDataAccessLevel acclvl)} how high is the access level.
+ * COUNTS_WITH_THRESHOLD < SUMMARY < MEASUREMENTS
+ */
+enum PatientDataAccessLevel {
+
+    /**
+     * Only access to patient counts, with resolution limited by the configured
+     * threshold. {@see Config}.
```

Comment:
which Config is that? 
maybe better to use `@link`

Reply:
we don't have this implemented yet. I'm going to remove the reference.